aws/table_aws_codepipeline_pipeline.go

```diff
@@ -0,0 +1,258 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codepipeline"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCodepipelinePipeline(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_codepipeline_pipeline",
+		Description: "AWS Codepipeline Pipeline",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([] string{"name", "region"}),
+			ShouldIgnoreError: isNotFoundError([]string{"PipelineNotFoundException"}),
+			Hydrate:           getCodepipelinePipeline,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCodepipelinePipelines,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the pipeline.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the pipeline.",
+				Hydrate:     getCodepipelinePipeline,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Metadata.PipelineArn"),
+			},
+			{
+				Name:        "created",
```

Comment:
Can we update the column to `created_at`?

Reply:
Updated the column name